holesky/.env.example

```diff
@@ -1,5 +1,10 @@
-ECDSA_PRIVATE_KEY_STORE_PATH=<path-to-your-ecdsa-private-key>
-BLS_PRIVATE_KEY_STORE_PATH=<path-to-your-bls-private-key>
-OPERATOR_BLS_KEY_PASSWORD=<bls-key-password>
-OPERATOR_ECDSA_KEY_PASSWORD=<ecdsa-key-password>
-DB_PATH=<path-to-a-directory-to-keep-operator-state>
```

Comment:
Removed all <path-to ...> and turned them into comments.

Is it possible that we shorten the variable names too? For example, for these 4 variables.
```
ECDSA_KEYSTORE_PATH=
ECDSA_KEY_PASSWORD=

BLS_KEYSTORE_PATH=
BLS_KEY_PASSWORD=
```

I tried to change them in this repository, but it seems that something needs to be done in the docker image.